### PR TITLE
Add MDX Storybook

### PR DESCRIPTION
**Pull request by jestapinski**
This PR adds a beginning MDX storybook! You can run this storybook the same way as the mitx storybook.

The MDX format changes how stories are written to something much simpler. There are `Icon` and `Color` stories here, but the `Icon` is the only one with mdx integrated.

To see the docs, click "docs" towards the top after `npm install`ing and running the storybook.

I also added a "Welcome" mdx story which can help document some storybook practices.

### CODE REVIEW

File: .storybook/config.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+import { action } from '@storybook/addon-actions';
+
+addParameters({
+    docs: {
+        container: DocsContainer,
+        page: DocsPage,
+        prepareForInline: storyFn => storyFn(),
+    },
+    options: {
+        storySort: (a, b) =>
```
Comment (jestapinski) [inline .storybook/config.js]: This is neat, it allows us to sort stories alphabetically

Comment (jbouzi12) [reply .storybook/config.js]: oh nice!

**Comment by jestapinski** [comment]
> Now I see them! Before the recent change I was seeing the welcome message for the colors story but things look good now 👍

Oh odd, thanks! I'll merge this in after the staging -> master.

**Comment by jestapinski** [comment]
Thanks all!

**Pull request merged**